functions/Restore-DbaDatabase.ps1

```diff
@@ -221,8 +221,30 @@ folder for those file types as defined on the target instance.
     }
     PROCESS
     {
-        
-        foreach ($f in $path)
+		$base = $SqlServer.Split("\")[0]
+		
+		if ($base -eq "." -or $base -eq "localhost" -or $base -eq $env:computername -or $base -eq "127.0.0.1")
+		{
+			# they can use whatever path they like
+		}
+		else
+		{
+			if ($Path.StartsWith("\\") -eq $false)
+			{
+				# Many internal functions parse using Get-ChildItem. 
```

Comment:
This will fail if the user passes a list of paths without using the pipeline (cause then it's an array).
Move this into the `foreach ($f in $path)` loop below. Add $IsLocal to the check, since you can't do if/else across begin/process.